test/bin/time-out

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# The Bash shell script executes a command with a time-out.
+# Upon time-out expiration SIGTERM (15) is sent to the process. If the signal
+# is blocked, then the subsequent SIGKILL (9) terminates it.
+#
+# Based on the Bash documentation example.
```

Comment:
This should address the license issue: https://github.com/ipfs/go-ipfs/pull/3152